modules/rosa-configure.adoc

```diff
@@ -11,7 +11,7 @@ Use the following commands to configure the {product-title} (ROSA) CLI, `rosa`.
 [id="rosa-login_{context}"]
 == login
 
-Log in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token from link:https://console.redhat.com/openshift/token/rosa[the {product-title} token page].
+Log in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token from link:https://console.redhat.com/openshift/token/rosa[the ROSA token page].
```

Comment:
Are these two actions (logging in and then saving), or are the credentials automatically going to a `rosa` kubeconfig?